### PR TITLE
Reword cancelled email

### PR DESCRIPTION
**Pull request by mikejolley**
Rewords from:

> "Alas. Just to let you know — [customer-name] has cancelled order [#order-number]:"

To:

> "Alas. Just to let you know — order [#order-number] belonging to [customer-name] has been cancelled:"

So it does not specific who cancelled it.

### CODE REVIEW

File: templates/emails/admin-cancelled-order.php [hunk 1]
```diff
@@ -24,8 +24,8 @@
 */
 do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
 
-<?php /* translators: %1$s: Customer full name. %2$s: Order number */ ?>
-<p><?php printf( esc_html__( 'Alas. Just to let you know &mdash; %1$s has cancelled order #%2$s:', 'woocommerce' ), esc_html( $order->get_formatted_billing_full_name() ), esc_html( $order->get_order_number() ) ); ?></p>
+<?php /* translators: %1$s: Order number, %2$s: Customer full name.  */ ?>
+<p><?php printf( esc_html__( 'Alas. Just to let you know &mdash; order #%1$s belonging to %2$s has been cancelled:', 'woocommerce' ), esc_html( esc_html( $order->get_order_number(), $order->get_formatted_billing_full_name() ) ) ); ?></p>
```
Comment (rodrigoprimo) [inline templates/emails/admin-cancelled-order.php]: Probably due to copy and paste error, `esc_html()` is being called inside another `esc_html()` call, resulting in a `PHP Warning:  printf(): Too few arguments` and an empty string. The same is happening in the other modified file.

**Review by rodrigoprimo — DISMISSED** [review]
I found a problem that is resulting in an empty string instead of the desired new string. I left an inline comment with more details.

Besides that, I noticed that the email subject is "[STORE_NAME] Test Customer has cancelled order #ORDER_ID". Maybe we should change it as well to not specify who canceled the order?

**Comment by mikejolley** [comment]
Thanks, I've adjusted for both points.

**Review by kloon — APPROVED** [review]
Just made a small adjustment to update the translators comment to reflect the string order. Good to merge now.

**Pull request merged**